scripts/checksum-device/checksum-device

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# checksum the first N bytes of a device
+# compare against expected result and return pass / fail
+# usage: checksum-device filepath size_of_filepath expected_sum
+
+set -eu
+
+infile=$1
+infile_size=$2
+expected=$3
+
+actual=$(dd if=$infile iflag=count_bytes count=$infile_size \
```

Comment:
I'd be extra careful with the quoting here because dd is scary :skull:  

`dd if="$infile" iflag=count_bytes count="$infile_size"`

Reply:
dd is scary.  Implemented.